src/tuples-and-arrays/tuples-and-arrays.md

```diff
@@ -13,22 +13,6 @@ types have a size fixed at compile time.
 | Arrays | `[T; N]`                      | `[20, 30, 40]`, `[0; 3]`          |
 | Tuples | `()`, `(T,)`, `(T1, T2)`, ... | `()`, `('x',)`, `('x', 1.2)`, ... |
 
-Array assignment and access:
-
-<!-- mdbook-xgettext: skip -->
-
-```rust,editable
-fn main() {
-    let mut a: [i8; 10] = [42; 10];
-    a[5] = 0;
-    println!("a: {a:?}");
-}
-```
-
-Tuple assignment and access:
-
-<!-- mdbook-xgettext: skip -->
```

Comment:
It looks like you removed this `<!-- mdbook-xgettext: skip -->` comment from this code block. Please restore the comment so that this code block is still skipped for localization.